src/app/shared/filter.service.ts

```diff
@@ -10,6 +10,8 @@ import { HttpService } from './http-service';
 
 import { FilterModel } from '../models/filter.model';
 
+import { WorkItem } from '../models/work-item';
```

Comment:
This is a direct dependency to WorkItem module, we don't need the type here, do we?